examples/notebooks/beam-ml/mltransform_basic.ipynb

```diff
@@ -0,0 +1,733 @@
+        "\n",
+        "To use `MLTransfrom`, install `tensorflow_transform` and the Apache Beam SDK version 2.50.0 or later.\n"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "!pip install tensorflow_transform --quiet\n",
+        "!pip install apache_beam>=2.50.0 --quiet"
+      ],
+      "metadata": {
+        "id": "MRWkC-n2DmjM"
+      },
+      "id": "MRWkC-n2DmjM",
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "88ddd3a4-3643-4731-b99e-a5d697fbc165",
+      "metadata": {
+        "id": "88ddd3a4-3643-4731-b99e-a5d697fbc165"
+      },
+      "outputs": [],
+      "source": [
+        "import apache_beam as beam\n",
+        "from apache_beam.ml.transforms.base import MLTransform\n",
+        "from apache_beam.ml.transforms.tft import ComputeAndApplyVocabulary\n",
+        "from apache_beam.options.pipeline_options import PipelineOptions\n",
+        "from apache_beam.ml.transforms.utils import ArtifactsFetcher"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "Artifacts are additional data elements created by data transformations. Examples of artifacts are the `minimum` and `maximum` values from a `ScaleTo01` transformation, or the `mean` and `variance` from a `ScaleToZScore` transformation. Look at https://beam.apache.org/documentation/ml/preprocess-data/#artifacts for more details on artifacts.\n",
+        "\n",
+        "\n",
+        "\n"
+      ],
+      "metadata": {
+        "id": "90nXXc_A4Bmf"
+      },
+      "id": "90nXXc_A4Bmf"
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "id": "bdabbc57-ec98-4113-b37e-61962f488d61",
+      "metadata": {
+        "id": "bdabbc57-ec98-4113-b37e-61962f488d61"
+      },
+      "outputs": [],
+      "source": [
+        "# store artifacts generated by MLTransform.\n",
+        "# Each MLTransform instance requires an artifact location to be empty.\n",
+        "# We use this method to delete and refresh the artifact location for each example.\n",
+        "artifact_location = './my_artifacts'\n",
+        "def delete_artifact_location(artifact_location):\n",
+        "  import shutil\n",
+        "  import os\n",
+        "  if os.path.exists(artifact_location):\n",
+        "      shutil.rmtree(artifact_location)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "id": "28b1719c-7287-4cec-870b-9fabc4c4a4ef",
+      "metadata": {
+        "id": "28b1719c-7287-4cec-870b-9fabc4c4a4ef"
+      },
+      "source": [
+        "## Compute and map the vocabulary\n",
+        "\n",
+        "\n",
+        "`ComputeAndApplyVocabulary` is a data processing transform that computes a unique vocabulary from a dataset and then maps each word or token to a distinct integer index. It facilitates transforming textual data into numerical representations for machine learning tasks.\n",
```

Comment:
Add link to transforms if possible, could be helpful for users to quickly take a look
```suggestion
        "[`ComputeAndApplyVocabulary`]() is a data processing transform that computes a unique vocabulary from a dataset and then maps each word or token to a distinct integer index. It facilitates transforming textual data into numerical representations for machine learning tasks.\n",
```